app/queries/invoices_query.rb

```diff
@@ -4,14 +4,14 @@ class InvoicesQuery < BaseQuery
   def call(search_term: nil, status: nil, filters: {}, customer_id: nil, payment_status: nil, payment_dispute_lost: nil, payment_overdue: nil) # rubocop:disable Metrics/ParameterLists
     @search_term = search_term
     @customer_id = customer_id
-    @filters = filters
+    @filters = filters = BaseQuery::Filters.new(filters)
```

Comment:
```suggestion
    @filters = BaseQuery::Filters.new(filters)
```

Reply:
This is a temporary step until this PR with query objects refactor gets merged (https://github.com/getlago/lago-api/pull/2313).

I would keep it this way to be able to use `filters` within the `call` method instead of change them all to `@filters` (other wise it will use the method `filters` param that is a hash) as the goal after the refactor would be use the filters from the new Query object design...

Reply:
I understand, thanks for the explanation.